### DISCUSSION TITLE
Librerias

### DISCUSSION THREAD

**Discussion by nicolass00** [ArisGuimera/JetpackComposePro#33] [General]
Tengo una pregunta, le van a agregar una seccion donde se muestren librerias que no existan de forma nativa en jetpack compose, como por ejemplo preferencias, ya que el otro dia estaba necesitando un settings screen y no encontre nada nativo, asi que pase un tiempo buscando alguna para poder hacer eso

**Comment by 006900** [comment]
de echo estaria bueno para que nos enteremos de componentes importantes como la libreria de accompanist,coil,glide,navigation compose y muchos otros

**Comment by ArisGuimera** [comment]
Justo lo estaba pensando pero no sé como organizarlo. ¿Una sección solo que sea de librerías? ¿O meter cada componente en su respectiva categoría pero con un aviso de que es una librería?

**Reply by 006900** [reply]
podria ser ... primero mostramos lo que hace la libreria , luego como funciona o como se utiliza y por ultimo la dependencia de la libreria

**Reply by ArisGuimera** [reply]
Y creamos una sección llamada librerías o por ejemplo coil que es para imágenes lo añadimos dentro de la sección de imágenes?

**Reply by 006900** [reply]
podria ser ya que tiene que ver con las imagenes ,asi como glide y pickaso que podrian estar en la seccion de imagenes ya que por defecto compose no nos trae un componente similar

**Comment by ArisGuimera** [comment]
> podria ser ya que tiene que ver con las imagenes ,asi como glide y pickaso que podrian estar en la seccion de imagenes ya que por defecto compose no nos trae un componente similar

Sí, parece que será lo mejor

**Reply by asnodj** [reply]
Hola gente ! Estoy aprendiendo Jetpack Compose y Kotlin. Ya llevo bastantes años en desarrollo web pero soy nuevo en mobile.
En mi aplicación quería poner un gráfico de tipo pie pero no encontré nada para hacerlo, solo hacerlo manualmente con canvas si bien recuerdo.

¿Hay librerías para eso? Si es el caso, entonces cuando se documente esa librería (si es que se hace) se deberá hacer una documentación con varios componentes de la misma (pie, barras, etc).

A tener en cuenta ;)
